15-Sprint-Mission/index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="icon" href="images/logo/favicon.ico">
+    <link rel="stylesheet" as="style" crossorigin="" href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css">
+ 
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="home.css">
+</head>
+<body>
+    <header>
+
+    <a href="/">
+        <img src="images/logo/panda-market-logo.png" alt="판다마켓홈" width="153">
+    </a>
+      <a href="login.html" id="loginLinkButton" class="button">로그인</a>
+    </header>
+    <main>
+      <section id="hero" class="banner">
+        <div class="wrapper">
+            <h1>
+                "일상의 모든 물건을"
+                <br>
+                "거래해 보세요"
+            </h1>
+            <a href="items.html" class="button pill-button">구경하러 가기</a>
+        </div>
+      </section>
+      <section id="features" class="wrapper">
+        <div class="feature">
+     
+          <img src="images/home/feature1-image.png" alt="인기 상품" width="50%">
```

Comment:
❗️ 수정요청

img 속성의 width 는 단위없는 정수여야합니다~ 위와 같이 작업하고 싶으시다면 css를 통해서 해주세요~
[mdn](https://developer.mozilla.org/ko/docs/Web/HTML/Element/img#width)

```suggestion
          <img src="images/home/feature1-image.png" alt="인기 상품">
```